frontend/package.json

```diff
@@ -10,6 +10,7 @@
   "type": "module",
   "dependencies": {
     "@apidevtools/swagger-parser": "^10.0.3",
+    "@dagrejs/dagre": "^1.1.2",
```

Comment:
Would it be possible to add the dependencies one commit at a time? This is useful for seeing which things are pulled in... but more importantly if it causes a side effect in some other part of the app (as can happen with maybe another dependency we have already being upgraded/downgraded) then we can bisect it more easily. If they are all in one commit this is harder.

Reply:
It used to be in a separate commit 

https://github.com/headlamp-k8s/headlamp/pull/2225#discussion_r1816316172

Reply:
@joaquimrocha see request and reasoning above for adding dependencies in separate commits.

Reply:
Discussed with Joaquim. We're going to leave it as is.